psql/functions/current_owner_has_organization_permission.sql

```diff
@@ -1,14 +1,13 @@
 CREATE FUNCTION app_hidden.current_owner_has_organization_permission(owner_uuid uuid, required_permission_slug text) RETURNS boolean AS $$
   SELECT EXISTS(
     SELECT 1
-    FROM team_permissions
-    INNER JOIN teams ON (team_permissions.team_uuid = teams.uuid)
-    WHERE teams.owner_uuid = current_owner_has_organization_permission.owner_uuid
-    AND team_permissions.owner_uuid = current_owner_uuid()
-    AND team_permissions.permission_slug IN (required_permission_slug, 'ADMIN')
+    FROM owner_permissions
+    WHERE owner_org_uuid = owner_uuid
+    AND owner_user_uuid = current_owner_uuid()
+    AND (permission_slug = required_permission_slug or permission_slug = 'ADMIN')
```

Comment:
You should avoid `OR` where possible for performance reasons; prefer `IN`:

Prefer
```suggestion
    AND (permission_slug IN (required_permission_slug, 'ADMIN'))
```